off-chain/test/Examples/Addition/Contract.purs

```diff
@@ -126,7 +126,7 @@ initialContract = do
   let
     lookups :: ScriptLookups.ScriptLookups AdditionValidator
     lookups = ScriptLookups.validator validator
-    datum = AdditionDatum $ { lockedAmount: BigInt.fromInt 100 }
+    datum = AdditionDatum $ { lockedAmount: BigInt.fromInt 1000 }
```

Comment:
Is there a special reason for this change?

Reply:
Hmm, not sure. Probably was testing something. Will put it back.

Reply:
nevermind I was asking because I thought it can matter.